x-pack/plugins/viz_editor/common/query_types.ts

```diff
@@ -8,13 +8,24 @@
 // Select clause
 // ----------------------------------------
 
+export type OperationName = 'column' | 'count' | 'date_histogram' | 'sum' | 'avg';
```

Comment:
OperationName is inaccurate, because 'eq' and 'lt' are operations, too. To stay consistent with the `BooleanOperator` defined below, maybe this should be `SelectOperator`?

Reply:
Done